cmd/apps/openfaas_app_test.go

```diff
@@ -22,3 +23,77 @@ func Test_getValuesSuffix_aarch64(t *testing.T) {
 		t.Errorf("suffix, want: %s, got: %s", want, got)
 	}
 }
+
+func Test_mergeFlags(t *testing.T) {
+	tests := []struct {
+		name  string
+		flags []string
+		want  map[string]string
+	}{
+		{
+			"openfaas",
+			[]string{
+				"nats.external.enabled=true",
+				"nats.external.host=openfaas-stan.openfaas.svc.cluster.local",
+				"nats.external.port=4222",
+				"nats.clusterName=\"faas-cluster\"",
+			},
+			map[string]string{
+				"nats.external.enabled": "true",
+				"nats.external.host":    "openfaas-stan.openfaas.svc.cluster.local",
+				"nats.external.port":    "4222",
+				"nats.clusterName":      "faas-cluster",
+			},
+		},
+		{
+			"issue #592",
+			[]string{
+				"controller.config.enable-opentracing='true'",
```

Comment:
All this detail is not required to prove that the one thing that was broken is fixed.

Less is more, reduce as much as possible so we can read this and understand what we're maintaining if we ever have to revisit it.